add_npm_package.sh

```diff
@@ -18,10 +18,7 @@ else
   PACKAGE_NAME=nodejs-${PACKAGE_MODULE}
 fi
 
-PACKAGE_DIR=$(compgen -G "packages/*/${PACKAGE_NAME}/")
-if [[ -z "${PACKAGE_DIR}" ]]; then
-  PACKAGE_DIR=packages/$BASE_DIR/$PACKAGE_NAME
-fi
+PACKAGE_DIR=$(compgen -G "packages/*/${PACKAGE_NAME}/") || PACKAGE_DIR=packages/$BASE_DIR/$PACKAGE_NAME
```

Comment:
```suggestion
PACKAGE_DIR=$(compgen -G "packages/*/${PACKAGE_NAME}/" || echo "packages/$BASE_DIR/$PACKAGE_NAME")
```

Does that work too?

Reply:
yeah, is it preferred?

Reply:
I generally prefer assigning a variable once if possible.

Reply:
sure, done